stable/zetcd/templates/deployment.yaml

```diff
@@ -29,6 +29,20 @@ spec:
 {{- else }}
             - "-endpoints" 
             - "{{ .Values.etcd.endpoints }}"
+{{- end }}
+{{- if .Values.etcd.tls }}
+{{- if .Values.etcd.tls.secret }}
```

Comment:
I'd call it `existingSecret` to make it explicit that it is not created by the chart. It's also used by other charts.

Do we really need the inner if statements?